support-frontend/assets/pages/[countryGroupId]/landingPage.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+	getAmountsTestVariant,
+	type Participations,
+} from 'helpers/abTests/abtest';
+import { Country } from 'helpers/internationalisation/classes/country';
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import { type GeoId, getGeoIdConfig } from 'pages/geoIdConfig';
+import { threeTierCheckoutEnabled } from 'pages/supporter-plus-landing/setup/threeTierChecks';
+import { ContributionsOnlyLanding } from 'pages/supporter-plus-landing/twoStepPages/contributionsOnlyLanding';
+import { ThreeTierLanding } from 'pages/supporter-plus-landing/twoStepPages/threeTierLanding';
+
+type Props = {
+	geoId: GeoId;
+	abParticipations: Participations;
+};
+
+const countryId: IsoCountry = Country.detect();
+
+export function LandingPage({ geoId, abParticipations }: Props) {
```

Comment:
I've had to add an extra wrapper component here because we need the country to figure out whether to show the contributions only landing page